packages/components/src/components/ui/SqErrorAlert.tsx

```diff
@@ -68,7 +68,7 @@ const StackTrace: FC<{ error: SqRuntimeError }> = ({ error }) => {
   ) : null;
 };
 
-export const SquiggleErrorAlert: FC<Props> = ({ error }) => {
+export const SqErrorAlert: FC<Props> = ({ error }) => {
```

Comment:
It's only used on ``SqErrors``, so this seemed appropriate

Reply:
`Sq` prefix is for squiggle-lang public classes, `Squiggle` prefix is for public components, confusingly we also use `Squiggle` in some internal components but not consistently.

I'm not sure what's the best convention here; we probably should drop `Squiggle` prefix for non-public components (as you did for `ValueMenuBar`).

But I also want to point out that this component is public.

It's exported in the top-level `index.ts`, and even documented in https://components.squiggle-language.com/?path=/docs/squiggle-squiggleerroralert--docs. (Which I think is correct, if `SquiggleViewer` for `SqValue` is public then users also need some way to render the second branch of `result<>`)

Which means that it's a bit inconsistent to have `SqErrorAlert` and `SquiggleViewer`; `SqErrorViewer` + `SqValueViewer` is one possible alternative.

Reply:
I'd be fine with ``SqValueViewer``. Would it break a lot of other stuff if I changed that? (Also want to change it to formally take ``settings`` as a prop instead of ``...settings``).

Reply:
> Would it break a lot of other stuff if I changed that?

It could break backward compatibility in versioned-components (because of [this](https://github.com/quantified-uncertainty/squiggle/pull/3005#pullrequestreview-1846717419)), but we don't use `SquiggleViewer` anywhere in the hub, so it should be fine.

> Also want to change it to formally take `settings` as a prop instead of `...settings`

I don't mind but this will be a small inconsistency if we don't nest settings in other public components (which we can't do easily because of versions).